core/src/main/scala/org/broadinstitute/dsde/rawls/submissions/SubmissionsService.scala

```diff
@@ -537,6 +545,10 @@ class SubmissionsService(
         ps.failureMode,
         ps.header
       )
+      _ <- getSetToDelete(workspaceName, submissionRequest)
+        .flatMap { setToDelete =>
+          entityServiceConstructor(ctx).deleteEntities(workspaceName, setToDelete.toSeq, None, None)
+        }
```

Comment:
It's not clear to me what would happen if two submissions coincided closely in time and tried to "fight" over the single attribute. It seems like this could happen with multiple users sharing a workspace, or API-based automation.